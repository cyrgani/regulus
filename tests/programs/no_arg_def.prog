_(
    def(hello, print("hello")),
    hello(),
    def(one, _(1)),
    assert_eq(one(), 1),

    print(catch(def(lit, 1))),
)