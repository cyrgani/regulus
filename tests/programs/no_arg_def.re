_(
    def(hello, print("hello")),
    hello(),
    def(one, _(1)),
    assert_eq(one(), 1),

    def(lit, 1),
    assert_eq(lit(), 1),
    assert(!(==(lit, 1))),
)