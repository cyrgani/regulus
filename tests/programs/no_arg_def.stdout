hello 
Assign: Error during definition: no valid function body was given! 
